src/main/groovy/com/netgrif/application/engine/petrinet/domain/dataset/logic/action/ActionDelegate.groovy

```diff
@@ -458,59 +474,89 @@ class ActionDelegate {
         }]
     }
 
-    protected SetDataEventOutcome createSetDataEventOutcome() {
-        return new SetDataEventOutcome(this.useCase, this.task.orElse(null))
+    def makeInterProcess(List<String> fieldIds, Closure behavior) {
```

Comment:
docstring should be added along with examples

Reply:
Added.